wmn-data.json

```diff
@@ -206,6 +206,16 @@
         "known" : ["GazetaPolska", "GPCodziennie"],
         "cat" : "social"
        },
+       {
+        "name" : "Allegro",
+        "uri_check" : "https://allegro.pl/uzytkownik/{account}",
+        "e_code" : 200,
+        "e_string" :"<title>Przedmioty użytkownika",

```

Comment:
I'm not sure that this `e_string` works. I tried the `Tefal_Polska` known account below and the title in the response was `<title>Oficjalny sklep Tefal - Strefa Marek Allegro</title>`. Can you please check this?